src/functions/accent.js

```diff
@@ -74,10 +74,14 @@ export const htmlBuilder: HtmlBuilderSupSub<"accent"> = (grp, options) => {
         // TODO(emily): Find a better way to get the skew
     }
 
+    const accentBelow = group.label === "\\c";
+
     // calculate the amount of space between the body and the accent
-    let clearance = Math.min(
-        body.height,
-        options.fontMetrics().xHeight);
+    let clearance = accentBelow
+        ? body.height + body.depth
+        : Math.min(
+            body.height,
+            options.fontMetrics().xHeight);
```

Comment:
The rendering looks good, but the fact that we have kern of size `-clearance` below is a bit confusing.  No action req'd since it's been like this for a while but something we might want to refactor in the future to be less confusing.

Reply:
I agree that it's confusing; I can't say I fully understand this code. Happy to review a PR that cleans it up. 🙂 There are some actual bugs here to fix, e.g., #1366 (though that has to do with horizontal rather than vertical spacing).